public/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "u-can-act",
+  "short_name": "u-can-act",
+  "start_url": ".",
+  "display": "standalone",
+  "background_color": "#fff",
+  "description": "Voorkom voortijdig schoolverlaten",
+  "icons": [{
+      "src": "/icons/Icon-192.png",
```

Comment:
maakt het hebben van een aparte /icons directory sense als we al een /images directory hebben (en als het favicon.ico zelf ook in /images zit)? Ik zou zeggen eventueel /images/icons, maar gewoon /images vind ik ook al wel prima. Maar icons zijn wel images denk ik.